js/src/components/google-combo-account-card/account-creation-description/account-info.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Display account info.
+ *
+ * @param {Object} props Props.
+ * @param {Object} props.account Account object.
+ * @param {string} props.text Text to display.
+ */
+const AccountInfo = ( { account, text } ) => {
+	if ( ! account?.id ) {
```

Comment:
@ankitrox We cannot rely solely on the account.id whether it's not falsy to show the account as connected. Please check my comment here: https://github.com/woocommerce/google-listings-and-ads/pull/2639/files#r1794227674 and let me know what you think.